eva_sub_cli/executables/cli.py

```diff
@@ -36,15 +36,9 @@ def validate_command_line_arguments(args, argparser):
         sys.exit(1)
 
 
-def get_version():
-    base_dir = os.path.abspath(os.path.dirname(main.__file__))
-    version = open(os.path.join(base_dir, 'VERSION')).read().strip()
-    return f'{version}'
-
-
-if __name__ == "__main__":
+def main():
```

Comment:
This conflicts with `main` which is imported:
```
Traceback (most recent call last):
  File "/home/april/projects/eva-sub-cli/venv/bin/eva-sub-cli.py", line 8, in <module>
    sys.exit(main())
  File "/home/april/projects/eva-sub-cli/venv/lib/python3.8/site-packages/eva_sub_cli/executables/cli.py", line 80, in main
    main.orchestrate_process(**args.__dict__)
AttributeError: 'function' object has no attribute 'orchestrate_process'
```

Reply:
Changed `main` -> `orchestrator`